code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -49,6 +49,13 @@
 		return FALSE
 	return TRUE
 
+/obj/item/ego_weapon/throw_at(atom/target, range, speed, mob/thrower, spin = TRUE, diagonals_first = FALSE, datum/callback/callback, force = MOVE_FORCE_STRONG, gentle = FALSE, quickstart = TRUE)
```

Comment:
This is not how you should do it, as it will invalidate any and all changes done to throwforce done after initialization.

Reply:
Additionally, you might want to put it in the obj/item, not just EGO weapons.

Reply:
Okay, looking at things, throwing code is a fucking mess throughout 3 files, and none of our ego weapons change throwing damage right now.
I'm not going to look through and do magic with variables for essentially a Hotfix.
It works, even if it doesn't work to your standards. I can remove this if you'd like.